packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -25,13 +56,30 @@ export const ChatList = (props: MgtTemplateProps & IChatListItemInteractionProps
   const { value } = props.dataContext as { value: GraphChat[] };
   const chats: GraphChat[] = value;
 
+  const chatListButtonItems = props.buttonItems === undefined ? [] : props.buttonItems;
+  const chatListMenuItems = props.menuItems === undefined ? [] : props.menuItems;
+  chatListMenuItems.unshift({
+    displayText: 'Mark all as read',
```

Comment:
Do we want this included by default within ChatList or move it to the containing app level?

Reply:
Seemed like Mark all as read should be taken care of by ChatList and not PlexChat? ChatList maintains its own ChatThreads and not PlexChat.